joplin/js/CreateContentModal/ButtonBar.js

```diff
@@ -1,29 +1,32 @@
-import React from 'react';
-import classNames from 'classnames';
+import React from "react";
+import classNames from "classnames";
 
-import './ButtonBar.scss';
+import "./ButtonBar.scss";
 
-const ButtonBar = ({ activeStep, handleBackButton, handleNextButton }) => (
-  <div className={classNames({
-    'ButtonBar': activeStep > 0,
-    'ButtonBar--hidden': activeStep === 0,
-  })}>
-    <div
-      className="ButtonBar__button"
-      onClick={handleBackButton}
-    >
+const ButtonBar = ({
+  activeStep,
+  handleBackButton,
+  handleNextButton,
+  handleCloseButton
+}) => (
+  <div
+    className={classNames({
+      ButtonBar: activeStep > 0,
```

Comment:
This isn't your code Brian, but I don't think we need this conditional so let's simplify the code. The 'ButtonBar' class should always show up. The only thing that needs to be conditional is the ButtonBar--hidden modifier.
```
className={classNames({
  "ButtonBar",
  "ButtonBar--hidden": activeStep === 0
})}
```